ush/python/pygfs/task/atm_analysis.py

```diff
@@ -82,11 +82,15 @@ def initialize(self: Analysis) -> None:
         jedi_fix_list = parse_j2yaml(jedi_fix_list_path, self.task_config)
         FileHandler(jedi_fix_list).sync()
 
-        # stage berror files
-        # copy static background error files, otherwise it will assume ID matrix
+        # stage static background error files, otherwise it will assume ID matrix
         logger.debug(f"Stage files for STATICB_TYPE {self.task_config.STATICB_TYPE}")
         FileHandler(self.get_berror_dict(self.task_config)).sync()
 
+        # stage ensemble files for use in hybrid background error
+        if self.task_config.DOHYBVAR:
+            logger.debug(f"Stage ensemble files for DOHYBVAR {self.task_config.DOHYBVAR}")
+            FileHandler(Analysis.get_ens_dict(self, self.task_config)).sync()
```

Comment:
```suggestion
            FileHandler(sel.f.get_ens_dict(self.task_config)).sync()
```
`atm_analysis.py` inherits from `analysis.py`, so this method is already in scope.

Reply:
Accept suggestion.  Done at 5e88371.